README.md

```diff
@@ -180,14 +217,13 @@ class PostData extends DataTransferObject
 }
 ```
 
-When PHP 7.4 introduces typed properties, you'll be able to simply remove the doc blocks and type the properties with the new, built-in syntax.
-
 ### Working with collections
 
 If you're working with collections of DTOs, you probably want auto completion and proper type validation on your collections too.
 This package adds a simple collection implementation, which you can extend from.
 
 ```php
+use \PostData;
 use \Spatie\DataTransferObject\DataTransferObjectCollection;
```

Comment:
Yeah this one should be fixed too, no one writes it that way
```suggestion
use Spatie\DataTransferObject\DataTransferObjectCollection;
```

Reply:
Going to resolve this conversation in the duplicate thread.